glutin/src/context.rs

```diff
@@ -97,6 +97,11 @@ pub trait PossiblyCurrentGlContext: Sealed {
     /// - **macOS: this will block if your main thread is blocked.**
     fn make_not_current(self) -> Result<Self::NotCurrentContext>;
 
+    /// Make the context not current to the current thread. If you need to
+    /// send the context toanother thread, use [`Self::make_not_current`]
```

Comment:
```suggestion
    /// send the context to another thread, use [`Self::make_not_current`]
```

Just noticed when decided to re-read.